### DISCUSSION TITLE
Link from OpenXml repo

### DISCUSSION THREAD

**Discussion by twsouthwick** [ShapeCrawler/ShapeCrawler#95] [General]
Hi - I'm a maintainer of the https://github.com/officedev/open-xml-sdk repo and have been adding a list of useful projects that build on OpenXml in the README. Would you mind if I add that there? If you want to submit a PR you can phrase the description how you'd like :)

**Comment by AdamShakhabov** [comment]
Wow! Hello. Would be great :) Just use the last one "A .NET library for manipulating PowerPoint presentations".
